src/modules/Dashboard/features/Participant/Activities/Activities.tsx

```diff
@@ -191,6 +191,7 @@ export const Activities = () => {
                   nickname: subject.result.nickname,
                   tag: subject.result.tag,
                   isTeamMember: subject.result.tag === 'Team',
```

Comment:
We should probably convert these tag-based checks to role-based wherever possible (I realize it can't be done everywhere yet):

```suggestion
                  isTeamMember: subject.result.roles.some((role) => TEAM_MEMBER_ROLES.includes(role))
```

Reply:
Good call. I'll update these ones